utils/generate_dotnet_channels.js

```diff
@@ -126,7 +126,7 @@ function objectType(props, indent, onlyOptional = false) {
   return { ts: `${indent}{${inner.ts}\n${indent}}`, scheme: `tObject({\n${inner.scheme}\n${indent}})` };
 }
 
-const yml = fs.readFileSync(path.join(__dirname, '..', 'src', 'protocol', 'protocol.yml'), 'utf-8');
+const yml = fs.readFileSync(path.join(__dirname, '../packages/playwright-core/src/protocol/protocol.yml'), 'utf-8');
```

Comment:
Keep it as before to work on Windows?

Reply:
afaik Node.js can deal with that. But let me revert it back to the old syntax.